src/generators/project/package.json.hbs

```diff
@@ -1,14 +1,15 @@
 {
   "name": "{{ projectName }}",
   "version": "0.1.0",
-  "repository": "https://github.com/CSSSR",
-  "author": "Andrew Laiff <q.laiff@gmail.com>",
   "private": true,
   "dependencies": {
-    "lerna": "^2.0.0"
+    "lerna": "^2.4.0"
   },
   "scripts": {
-    "bs": "lerna bootstrap",
-    "postinstall": "ris postinstall"
-  }
+    "bs": "yarn && ris clean-cache",
```

Comment:
Why we dont use `lerna bootstrap` here?

Reply:
`lerna bootstrap` crashed, without error output, if use `yarn workspaces'
but, `yarn install` works fine